mailhandler/spameater

```diff
@@ -421,13 +425,15 @@ $config->debug('actually not... the account is disabled.');
       $for =~ /(.*)\@/;
       # if there was no special syntax (i.e., this is just a plain address)
       if ($musername eq $1) {
-        &sendMail($config,$useXHeader,$RealEmail,\$msg,'');  # just deliver the message
-        next;                      # and skip the rest of the loop 
+        &sendMail($config,$useXHeader,\@rcpts,\$msg,'');  # just deliver the message
+        next;                      # and skip the rest of the loop
       }
     }
   }
   $UserID = 0 if !$UserID || $util->hasFeature('ACCOUNTDISABLED',$Features);
 
+  @rcpts = ($RealEmail);
+  @spamrcpt = ($SpamEmail);
```

Comment:
If these lines are excluded, the diff size would be smaller. Doing some quick checking, this is redeclaring the real/spam emails under new array variables with only one item.